docker/development/.env.postgres

```diff
@@ -0,0 +1,3 @@
+POSTGRES_USER=user
```

Comment:
Since we never expect users to change this when testing I would just hardcode all 3 env vars in the compose file (but maybe use `newdle` instead of `user`

Reply:
now it's used in both the docker-compose files